server/src/main/java/io/crate/replication/logical/MetadataTracker.java

```diff
@@ -522,6 +508,12 @@ private ClusterState removeDroppedTablesOrPartitions(String subscriptionName,
                     relations.put(relationName, state);
                 }
             }
+            updatedClusterState = DropSubscriptionAction.removeSubscriptionSetting(
```

Comment:
I guess this is for stopping tracking of those relations? Can we add tracker not alive check in the end of the test?

Reply:
No the tracking of the publication will continue.